src/test/java/org/opendevstack/provision/services/CleanupAtlassianProjectsMT.java

```diff
@@ -20,6 +20,7 @@
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+import org.springframework.boot.test.mock.mockito.MockBean;
 import org.springframework.core.env.Environment;
 import org.springframework.test.context.ActiveProfiles;
 
```

Comment:
We should add to the class name the suffix `Test`

Reply:
Do you know by any chance what MT in the class name stands for?

Reply:
looking at them it looks like it could mean 'ManualTest'

Reply:
then let us add as suffix `ManualTest`. The name will be then more expressive.

Reply:
done